charts_flutter/lib/src/canvas/circle_sector_painter.dart

```diff
@@ -64,19 +64,15 @@ class CircleSectorPainter {
         endAngle != null &&
         endAngle - startAngle == 2 * pi;
 
-    final midpointAngle = (endAngle + startAngle) / 2;
-
     final path = new Path()
       ..moveTo(innerRadiusStartPoint.x, innerRadiusStartPoint.y);
 
     path.lineTo(radiusStartPoint.x, radiusStartPoint.y);
 
     // For full circles, draw the arc in two parts.
```

Comment:
I'd suggest removing these comments, "// For full circles, draw the arc in two parts" here and below. This will no longer be true.

Reply:
Completely agree. I've just updated PR. Thank you.